Web Server/conf/conf.d/52_webapp_plugin.conf

```diff
@@ -1,6 +1,6 @@
-<Directory htdocs/plugins/webapp_plugin/resources>
+<Directory htdocs/plugins/webapp_plugin>
     # Tell the privilege module to use webapp_plugin.htpriv
 	Session On
     AuthNIPrivilegeApplication webapp_plugin
-    Require privilege ModifyResource
+    Require privilege ViewResource
```

Comment:
Unfortunately the lvdist configuration for building ni packages doesn't look like it has the ability to run scripts after install. If there are not major advantages (ie participation in org modeling, etc) it seems easier for a WebVI developer / lvdist-based nipkg consumer to use the conf pattern.

I can make a follow-up PR to switch to the modernized conf pattern unless there are other concerns.

Reply:
Ah, I didn't realize this was intended to be deployed via package instead of manually copying files. Even with the conf file, they would need to know to restart the web server before it takes effect. Do you know if/where NI Package Builder might fit into this? Can it create packages from a web VI? I would assume it supports custom actions.

Reply:
Adding or modifying the example to use NI Package Builder may be useful. I'll make a PR for the minor changes to the current workflow discussed here. Looking more into NI Package Builder seems like a useful follow-on change.